src/compute/commitments.rs

```diff
@@ -110,6 +111,50 @@ pub fn compute_commitments_with_generators(
     }
 }
 
+#[doc = include_str!("../../docs/commitments/compute_commitments_with_generators.md")]
+///
+///# Example 1 - Pass generators to Commitment Computation
+///```no_run
+#[doc = include_str!("../../examples/pass_generators_to_commitment.rs")]
```

Comment:
Why are you referencing these examples? Aren't those for curve-25519?

Reply:
Yes, they do refer to curve-25519. I am referencing those for two reasons. 

1. The `doc` tag doesn't seem to be working, see [PROOF-708](https://space-and-time.atlassian.net/browse/PROOF-708). I want to get that fixed before I write the documentation. I'm not positive how to test formatting without understanding how the `doc` tag should appear. For example, I don't know if I can use the same `md` file to document both functions, or need a separate `md` file?
2. If I remove the `doc` tags, `cargo` throws an error: `error: missing documentation for a function`. 

Here is how the `doc` tag appears now when hovering over a function.
![image](https://github.com/spaceandtimelabs/blitzar-rs/assets/6509385/14e7081e-1ab3-4df5-9331-f5b9cfdba924)

And `blitzar` doesn't have a Documents section on crates.io.
![image](https://github.com/spaceandtimelabs/blitzar-rs/assets/6509385/66ce8041-e5ea-429a-aaf7-41ae7de28925)

I was hoping to avoid scope creep on this PR.

Reply:
If you need the tags as a placeholder, please indicate so with a comment so that someone looking at the code understand why they're there.

Reply:
Great idea. Added.